en/dev_log/log_encryption.md

```diff
@@ -0,0 +1,179 @@
+# Log Encryption
+
+The [System Logger](../modules/modules_system.md#logger) can be used to create encrypted logs, which may then be decrypted manually before analysis.
```

Comment:
What is the first version of PX4 in which this is supported?

Reply:
I know it works on 1.14. 

I am unsure about specifics about the first version where this functionality is supported and is stable. I think [this PR ](https://github.com/PX4/PX4-Autopilot/pull/17814) added it all the way back in 2021, but there were other PRs since then by @jlaitine related to it.  

I did some commit searches and I think 1.12.0 is the earliest version where the crypto commits exist, but I am unsure how it works on that version, since I only tested it on 1.14.

Reply:
Thanks. The parameter appears in v1.14 first, so I have added a badge for that.